### DISCUSSION TITLE
急急急！！！调试，是不是要建立launch.json文件配置调试？如何快速建立这个文件？

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#895] [Q&A]
vscode中有快速建立launch.json配置文件，相比nvim中也有插件支持吧，大佬，，知道是哪个吗？或者告诉我使用方法

**Comment by Jint-lzxy** [comment]
Did u read through [`:h dap.txt`](https://github.com/mfussenegger/nvim-dap/blob/master/doc/dap.txt)? Usually nvim-dap accepts Lua tables for user configuration; to use `launch.json`, you need to invoke the extension provided by nvim-dap but you still need to understand how things work to properly code that file (which is slightly different from VSCode). Unfortunately, you will need to code this yourself or make changes from the preset provided by upstream specifications (if any). We have several presets available at [configs/tool/dap/clients](https://github.com/ayamir/nvimdots/tree/main/lua/modules/configs/tool/dap/clients), which should meet most needs to debug.

See also:
https://github.com/ayamir/nvimdots/blob/6de8afe0fc4a6e53a9d64a013327d5482b6dbe99/lua/core/settings.lua#L107-L115

**Reply by wxzvim** [reply]
okok，，感谢，，看了帮助，，全是英语，，读起来比较费劲，，有时间再看，，现在就要用

**Reply by wxzvim** [reply]
向大佬学习，，回答都用英语，，可以可以，，这种方式锻炼英语，，不错不错

**Reply by wxzvim** [reply]
发现帮助文档里没有哈
是不是要加nvim-dap-pyhton这个插件

**Reply by Jint-lzxy** [reply]
So what exactly do you want to achieve? Your title conflicts with your reply.

Please always try to avoid asking questions that involve [The XY Problem](https://xyproblem.info/).

**Reply by charliie-dev** [reply]
classic vscode addict, always want to do things in vscode way

**Comment by Jint-lzxy** [comment]
Closed due to inactivity. Please reactivate if further support is needed.
